src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Container, Divider } from "@mui/material";
+import { LEGAL_LINKS } from "../../constants/LegalLinks";
+import { QUICK_LINKS } from "../../constants/QuickLinks";
+import Newsletter from "../Newsletter";
+import FooterBottom from "./FooterBottom";
+import FooterLinksGroup from "./FooterLinksGroup";
+import LogoSection from "./LogoSection";
+
+const Footer: React.FC = () => {
+  return (
```

Comment:
you don't need a return statement in arrow function, when you have 1 item to return :)